src/control/controlbehavior.cpp

```diff
@@ -161,19 +171,19 @@ double ControlAudioTaperPotBehavior::valueToParameter(double dValue) {
     } else if (dValue < 1.0) {
         // db + linear overlay to reach
         // m_minDB = 0
-        // 0 dB = m_neutralParameter
+        // 0 dB = m_dNeutralParameter
         double overlay = m_offset * (1 - dValue);
         if (m_minDB) {
-            dParam = (ratio2db(dValue + overlay) - m_minDB) / m_minDB * m_neutralParameter * -1;
+            dParam = (ratio2db(dValue + overlay) - m_minDB) / m_minDB * m_dNeutralParameter * -1;
```

Comment:
Yeah I noticed that, but it happens that it makes sense for both m_dNeutralParameter and m_neutralParameter to be the same, because we definitely want the parameter that hides the ring completely to be the same that produces 0db gain.

Maybe in the future we discover a specific CO for which the two values shall not be the same, but I'll not add support for this now because we don't need it yet. My motto: "DECUN: Don't Even Code it Until it's Needed." :)

> Do you use a refactoring tool? This will perform all renaming automatically.

What's that? I have the search&replace tool in my beloved geany :)